docs/dasharo-tools-suite/overview.md

```diff
@@ -24,7 +24,35 @@ Thank you for using Dasharo Tools Suite. If you have encountered any problems
 with this system or would like to provide feedback for us - please open an issue
 on [Dasharo
 issues](https://github.com/Dasharo/dasharo-issues/issues?q=is%3Aopen+is%3Aissue+label%3ADasharoToolsSuite).
+Dasharo Tools Suite has its own label: `DasharoToolsSuite` and issue template
+for bugs: `DTS bug report`. Other types of issues in the repository are generic
```

Comment:
Simply provide a direct link to the report: https://github.com/Dasharo/dasharo-issues/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml and instruct to choose `Dasharo Tools Suite` as a component

Reply:
Link added here b07c88d5d094dcab8b4de9bbec664d15942e09fc.